### DISCUSSION TITLE
Looks like ONC exploits are patched in ChromeOS 129

### DISCUSSION THREAD

**Discussion by QinCai-rui** [3kh0/ext-remover#1269] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Yesterday I was using CrOS 127 and the ONC exploits were working fine. I am currently using 129, and it appeared to be working fine when I uploaded the ONC file to chrome://network/#general. However, it doesn't seem to be actually uploaded as when I refreshed the page, the "Networks imported: 1" at the bottom of the page just disappeared. I am thinking that it might be possible to spam upload the file maybe? Further investigation needed.

**Comment by akabutnicer** [comment]
uh so before you go casting theories, I don't know, maybe check if you can update the Chromebook... 🥱

**Reply by QinCai-rui** [reply]
sorry I am not sure what you mean 😭

**Reply by S-PScripts** [reply]
> sorry I am not sure what you mean 😭

He said that you should check that if you can update your chromebook, in case it's just v129 with some sort of bug and v130 fixed it

**Reply by akabutnicer** [reply]
👍

**Reply by akabutnicer** [reply]
also after testing this on 128, the same did happen, but I was **not** allowed to update my Chromebook. so in summary, I doubt this exploit is patched

**Reply by must9f** [reply]
Doesn't work for me on v130 or v129

**Reply by akabutnicer** [reply]
weird. i still doubt it's the version's fault though

**Reply by akabutnicer** [reply]
does cauDNS work?
cauDNS: https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/CauDNS

**Reply by must9f** [reply]
doesn't work on 126 for me, not the version. do you know any other goguardian exploits for 126?

**Reply by akabutnicer** [reply]
i don't know maybe rigtools?

**Reply by must9f** [reply]
I tried rigtools and none of the buttons work for me, might just be dumb or it doesn't work for my district

**Reply by sites1235** [reply]
> I tried rigtools and none of the buttons work for me, might just be dumb or it doesn't work for my district

Because you don't have those extensions.

**Comment by schoolexploitkid** [comment, marked as answer]
it should be patched, it doesnt work for me

**Reply by jeffrey6576i** [reply]
yo @schoolexploitkid do you know how to do droneblocks

**Reply by S-PScripts** [reply]
what

**Reply by akabutnicer** [reply]
what

**Reply by jeffrey6576i** [reply]
i meant like how do i do the droneblocks method

**Reply by akabutnicer** [reply]
what
